### ISSUE TITLE
Generate Eloquent Helper Code

### ISSUE THREAD

**Issue by K2ouMais** [laravel-idea/plugin#56]
First I want to thank you for this.

I was waiting for this, for so long time...

It works like you said, but for some reasons it doesnt find the propertys of some models.

A method that it also doesnt find is "orderBy".

With the normal IDE-Helper "orderBy" is available.

I will test more tomorrow and give some more feedback.

**Comment by adelf** [comment]
Yes. It's experimental, so I'm sure there will be a lot of issues with it. Tomorrow I'll release a new version with some fixes already )
Please, share all issues you will catch )

**Comment by K2ouMais** [comment]
Tomorrow another version?? You are awesome.

Just because of this, is the reason why I dont regret buying your tool.

**Comment by adelf** [comment]
Hi. 3.0.3 version generator should add `orderBy*` methods and many others.

**Comment by K2ouMais** [comment]
@adelf 
Yes i can confirm, that it is working.
Next please the propertys.

Thanks

**Comment by adelf** [comment]
Could you share the cases when plugin didn't fetch the property?

**Comment by K2ouMais** [comment]
Here is a small Model:

```php
namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class AccreditedCountry extends Model
{
    protected $connection = 'ABC';
    protected $table = 'ABC_Accredited_Countries';

    public static function allAccreditedCountries()
    {
        return self::orderBy('accredited_countries_name')->paginate(50));
    }

    public static function aAccreditedCountry($country)
    {
        return self::where('accredited_countries', $country)->paginate(50);
    }

    public static function isAccreditedCountry($country)
    {
        if (self::where('accredited_countries', $country)->count() === 1) {
            return true;
        }

        return false;
    }
}

```

For this Model as example I only have this in the ```_ide_helper_models.php```:

```php
     * @property AccreditedCountry $accreditedcountry
```

Somehow I was expecting to see the names of all columns as a property.

**Comment by adelf** [comment]
Plugin fetches information about table fields from migrations. Do you have migrations for `ABC_Accredited_Countries` table in the project?

**Comment by K2ouMais** [comment]
That could be a problem, because I am using an external Database and I dont have migrations for that.

How do IDE-Helper get the properties?

**Comment by adelf** [comment]
It fetches column right from database. I'll add fetching from database in one of the next versions.

**Comment by K2ouMais** [comment]
That would be great, because not everyone is using migrations.

Thank you

**Comment by K2ouMais** [comment]
Have you an estimated date, when your plugin will fetch columns directly from the database?

**Comment by adelf** [comment]
Let me make some investigation...

**Comment by adelf** [comment]
The next 3.2 version will contain fetching table and column data from database.

**Comment by K2ouMais** [comment]
Awesome work. Thank you

**Comment by adelf** [comment]
Implemented in the 3.2 version. Please write here if I missed something.
